### PR TITLE
TR-2224 Improve documentation for scheduled transmissions

### PR DESCRIPTION
**Pull request by mtgray-sp**

### CODE REVIEW

**Comment by ndurant-sp** [comment]
So I'm torn on whether we should do the migration level updates and hold this PR until we start migration, or do this small update now and another before migration.  Thoughts?

File: content/api/transmissions.apib [hunk 1]
```diff
@@ -638,6 +638,16 @@ Only transmissions which are scheduled for future generation may be deleted.
 
 Delete all transmissions in a campaign.
 
+Please note that this endpoint immediately returns a 204 response and the
+deletion process happens in the background. When a message is deleted, you will
+receive a bounce event with a reason of "554 5.7.1 [internal] Campaign
+cancelled".
```
Comment (ndurant-sp) [inline content/api/transmissions.apib]: I would put a comment that you can see these via Events API or through Webhooks.

**Comment by ndurant-sp** [comment]
Should remove the list by campagin ID since it isn't supported.

**Comment by mtgray-sp** [comment]
> So I'm torn on whether we should do the migration level updates and hold this PR until we start migration, or do this small update now and another before migration. Thoughts?
> 
> If we are going to hold this until migration, we should remove the GET docs, delete by transmissions id, the enterprise banners around delete by campaign, and probably others.

I'm leaning towards shipping this small change now because I think it could be useful to customers. I can update the A/C of TR-2147 to make sure we capture the changes you listed.

**Comment by ndurant-sp** [comment]
> > So I'm torn on whether we should do the migration level updates and hold this PR until we start migration, or do this small update now and another before migration. Thoughts?
> > If we are going to hold this until migration, we should remove the GET docs, delete by transmissions id, the enterprise banners around delete by campaign, and probably others.
> 
> I'm leaning towards shipping this small change now because I think it could be useful to customers. I can update the A/C of TR-2147 to make sure we capture the changes you listed.

I'm good with that.  Lets ship this out and we'll document the other pieces as part of migration.

**Pull request merged**